internal/common/error.go

```diff
@@ -158,16 +158,26 @@ func (t *codeError) Is(err error) bool {
 	return ok
 }
 
-func IsRetryErrorCode(code commonpb.ErrorCode) bool {
+func GetErrorCode(e error) (commonpb.ErrorCode, bool) {
+	codeErr, ok := e.(*codeError)
+	if !ok {
+		return -1, false
+	}
+	return codeErr.code, true
+}
+
+func IsRetryableErrorCode(code commonpb.ErrorCode) bool {
```

Comment:
I thought list DoNotRetryError should be much easier than list Retriable errors. Because there maybe many errors we don't know whether we should retry and we can always retry on it.

Reply:
This could be fixed in later version with correct error code